packages/manager/modules/manager-pci-common/src/translations/rclone-download/Messages_fr_CA.json

```diff
@@ -0,0 +1,14 @@
+{
+  "pci_projects_project_users_download-rclone_title": "Télécharger un fichier de configuration rclone",
+  "pci_projects_project_users_download-rclone_submit_label": "Télécharger",
+  "pci_projects_project_users_download-rclone_cancel_label": "Annuler",
+  "pci_projects_project_users_download-rclone_content": "Vous allez télécharger un fichier de configuration permettant de vous connecter avec rclone.",
+  "pci_projects_project_users_download-rclone_more_link": "En savoir plus",
+  "pci_projects_project_users_download-rclone_region_label": "Sélectionner la région dans laquelle ce fichier s'appliquera :",
+  "pci_projects_project_users_download-rclone_region_label_global": "Sélectionner la région dans laquelle ce fichier s'appliquera (seules les régions proposant les services Object Storage et Cloud Archive sont listées) :",
+  "pci_projects_project_users_download-rclone_region_help": "Pour des raisons de sécurité le fichier de configuration ne contient pas votre mot de passe. Il est donc nécessaire d'éditer le fichier de configuration pour y ajouter manuellement votre mot de passe.",
+  "pci_projects_project_users_download-rclone_success_message": "Le téléchargement du fichier de configuration rclone a commencé, si ce n'est pas le cas ",
+  "pci_projects_project_users_download-rclone_success_message_link": "cliquez-ici",
```

Comment:
```suggestion
  "pci_projects_project_users_download-rclone_success_message_link": "cliquez ici",
```

Reply:
it comes from angulars js : https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/components/users/download-rclone/translations/Messages_fr_FR.json

we won't submit a new translation request here, if there is an issue with those translations used in production it can be handled in the scope of run